templates/full-time-program.html

```diff
@@ -18,10 +18,9 @@ <h2>
     <div class="column">
       <h1 class="blue-h1">The Program</h1>
       <p class="justify">
-        Techtonica's full-time software engineering training program is a 
-        virtual, close to year-long program that provides the opportunity for
-        women and non-binary adults seeking economic empowerment to learn technical skills in
-        a safe, inclusive space and start their first jobs in tech. We select
+        Techtonica provides the opportunity for women and non-binary adults 
```

Comment:
Sorry, could we expand this a little to "Techtonica’s full-time, virtual software engineering training program provides the opportunity . . ."

Reply:
### ✅ Addressed in [f5a0daf](https://github.com/Techtonica/techtonica.org/pull/336/commits/f5a0daf59b8f915df48c3b6f5763dc25ed05dbd7).